learningequality/ka-lite
# Video update update

Fixing up the video update page, by:
- Merging in changes lost in our release-0.10 / develop divergence
- Cleaning up shift to the "updates" app
- Defining a set of test scenarios, and testing them.  Manually.  Many times :)
- Along the way, fixing #475

Scenarios to test:
1. Download one video.
2. Download two videos.
3. Delete a video, then re-download the video.
4. Download a video.  When that completes, download another video.
5. Download many videos.  While downloading, leave the page and come back.
6. Download many videos.  While downloading, open a new tab and go to the update video page.
7. Download many videos.  Click "cancel" after downloading begins.
8. Download many videos.  While downloading, delete videos.

Things to check:
1. Videos are downloaded properly
2. Progress bar updates properly
3. Half-circle is filled for current video downloading
4. Full-circle appears after current video is downloaded

@jamalex, need you to review code, as much of the update video changes that had to be merged over were yours.
@wangguan59, @rtibbles if you could test, that would be great!
